spec/decision_service_spec.rb

```diff
@@ -120,11 +126,12 @@
     end
 
     it 'should bucket normally if user is whitelisted into a forced variation that is not in the datafile' do
+      expect(config.set_forced_variation('test_experiment', 'forced_user_with_invalid_variation', 'invalid_variation')).to eq(FALSE)
       expect(decision_service.get_variation('test_experiment', 'forced_user_with_invalid_variation')).to eq('111128')
       expect(spy_logger).to have_received(:log)
                             .once.with(
                               Logger::INFO,
-                              "User 'forced_user_with_invalid_variation' is whitelisted into variation 'invalid_variation', which is not in the datafile."
+                              "User 'forced_user_with_invalid_variation' is in variation 'control' of experiment 'test_experiment'."
```

Comment:
please discard it. it's whitelist case not the forced variation case.

Reply:
Discarded this change